### PR TITLE
{Compute} `az vmss`: Migrate commands to Code Gen V2

### PR DESCRIPTION
**Pull request by yanzhudd**
**Related command**

`az vmss delete/get-os-upgrade-history/list/list-instances/list-skus/perform-maintenance/simulate-eviction`

**Description**

Migrate `az vmss` commands:

`az vmss delete/get-os-upgrade-history/list/list-instances/list-skus/perform-maintenance/simulate-eviction`

Some commands in `az vmss` group are not migrated yet:

`az vmss create\deallocate\delete-instances\get-instance-view\list-instance-connection-info\list-instance-public-ips\reimage\restart\scale\show\start\stop\update\update-instances\wait\set-orchestration-service-state`

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Compute

**Comment by yanzhudd** [comment]
> ❌AzureCLI-BreakingChangeTest

It has been verified with no breaking change.

File: src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_get_os_upgrade_history.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss get-os-upgrade-history",
+)
+class GetOsUpgradeHistory(AAZCommand):
+    """List list of OS upgrades on a VM scale set instance.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_get_os_upgrade_history.py:18]:
```suggestion
    """List the OS upgrades on a VM scale set instance.
```

File: src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_list.py [hunk 2]
```diff
@@ -0,0 +1,2282 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss list",
+)
+class List(AAZCommand):
+    """List a list of all VM scale sets under a resource group.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_list.py:18]:
```suggestion
    """List all VM scale sets under a resource group.
```

File: src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_list_instances.py [hunk 3]
```diff
@@ -0,0 +1,1422 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss list-instances",
+)
+class ListInstances(AAZCommand):
+    """List a list of all virtual machines in a VM scale sets.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_list_instances.py:18]:
```suggestion
    """List all virtual machines in a VM scale sets.
```

File: src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_list_skus.py [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss list-skus",
+)
+class ListSkus(AAZCommand):
+    """List a list of SKUs available for your VM scale set, including the minimum and maximum VM instances allowed for each SKU.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_list_skus.py:18]:
```suggestion
    """List SKUs available for your VM scale set, including the minimum and maximum VM instances allowed for each SKU.
```

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**